### DISCUSSION TITLE
Cannot ALTER SYSTEM anymore?

### DISCUSSION THREAD

**Discussion by Eldynn** [supabase/supabase#15360] [Questions] [answered]
I have this use case where I would like to ALTER SYSTEM to update a variable in local, staging and production so my database webhook can use it to send to the correct url depending on the environment.

It seem the new version of Supabase does not allow the dashboard and migration user to ALTER SYSTEM.

Is it possible to it in another way? Via the UI? Another command?

**Comment by GaryAustin1** [comment, marked as answer]
Can you just use Vault or a private schema with a settings table to do this?
I assume you are caught up in this https://github.com/orgs/supabase/discussions/9314 as superuser is no longer available.

**Reply by Eldynn** [reply]
Thank you, exactly what I was looking for!
